themes/gitline/gitline.theme.bash

```diff
@@ -64,5 +64,8 @@ THEME_CLOCK_FORMAT=${THEME_CLOCK_FORMAT:="%H:%M:%S"}
 IN_VIM_THEME_PROMPT_COLOR=245
 IN_VIM_THEME_PROMPT_TEXT="vim"
 
+IN_TOOLBOX_THEME_PROMPT_COLOR=125
+IN_TOOLBOX_THEME_PROMPT_TEXT=" "
```

Comment:
Clever use of the [Font-Awesome Wrench](https://fontawesome.com/v4.7.0/icon/wrench) :)  (or perhaps one was already based on the other?)

Since the documentation for this theme explicitly mentions it being designed for the [DroidSansMono Nerd Font](https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/DroidSansMono), have you confirmed this character is available there?  It should be,  since it shows up on the [Nerd Font Cheat Sheet](https://www.nerdfonts.com/cheat-sheet) (code 'nf-fa-wrench'), but you may want to confirm for completeness.

Reply:
I didn't try installing the font myself, but I did download the .otf file, uploaded to http://fontdrop.info, and checked how the character looks like. I couldn't find the same icon as the toolbox project (a wrench in hexagon), so I just used a wrench character.